src/styles/mixins/mixins.scss

```diff
@@ -64,3 +66,11 @@
     @content;
   }
 }
+
+@mixin media-query($sizes...) {
+  @include media($sizes...) {
+    @supports (display: grid) {
```

Comment:
Are you sure all of our other browsers support grid and the @supports tag?

Reply:
Yes